content/docs/secure/encryption/rekey.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Rekey Configuration"
+linkTitle: "Rekey Configuration"
+weight: 3
+Description: >
+  Rekey Configuration and Usage
+---
+
+## Rekey Controller Installation
+
+The CSM Encryption Rekey Controller must be installed in order to be able to perform key rotation on CSM encrypted volumes in a
+Kubernetes cluster. The Rekey Controller can be installed via the standard Dell Helm Chart repository available at  https://github.com/dell/helm-charts.git.
+
+A secret of the cluster config must be created with the name ``cluster-kube-config`` typically from the .kube/config. Here is an example:  
+
+```
+kubectl create secret generic cluster-kube-config --from-file=/home/root/.kube/config
+```
+
+The `csm-encryption-rekey-controller/values.yaml` must be adapted to the kubernetes cluster via the `provisioner` and `port` parameters.
+Here is an example of a typical values.yaml file:
+
+```yaml
+# Rekey controller image name.
+image: dellemc/csm-encryption-rekey-controller:v0.1.0
+
+# Rekey controller image pull policy.
+# Allowed values:
+#  Always: Always pull the image.
+#  IfNotPresent: Only pull the image if it does not already exist on the node.
+#  Never: Never pull the image.
+imagePullPolicy: IfNotPresent
+
+# This value is required and must match encryption.pluginName value
+# of the corresponding Dell CSI driver.
+provisioner: sec-isilon.dellemc.com
+# This represents the port number used by encryption driver Rest API server.
+port: 3838
+```
+> **NOTE**: It is important that the `provisioner` reflect the same name as the `pluginName` specified in the standard encryption values.yaml file, and the `port`
+reflect the same port number as the encryption driver REST API server would be listening on. If they don't match then all rekey attempts will fail.

```

Comment:
Instead of this, standard practice in csm-docs is to place a table with Parameter,Description,Required,Default. 
For `provisioner` the description should be just: "This value is required and must match encryption.pluginName value of the corresponding Dell CSI driver."
For `port` the description should be just: "This value is required and must match encryption.apiPort value of the corresponding Dell CSI driver."
image, imagePullPolicy and logLevel - not required and have default values
provisioner and port - required and have no default values

Reply:
adopting a table format as recommended.